WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2322,3 +2322,19 @@ PrefOOOExternalMsg          = margin-top:@PrefViewGutterSpace@;
 PrefOOOTimePeriodCell       = padding:@PrefSectionInternalSpacing@ 5px;
 PrefOOOMiniDropDown         = @PrefRowMiniSelectWidget@;
 PrefOOODurationLabel        = text-transform:lowercase;
+
+######################################
+#   PREFERENCES -Mail Page
```

Comment:
We can use either page or section.
I used page because the file name of script files having Preference -- mail is ZmMailPrefsPage and likewise for others.